cdap-unit-test/src/test/java/io/cdap/cdap/spark/SparkTest.java

```diff
@@ -164,6 +164,7 @@ public void testClassicSpark() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
For tests that we have added an ignore tag, can we go ahead and open a JIRA ticket for them and add as a TODO comment? Preferably logically grouped by efforts it would take to fix. Thanks!

Reply:
Ack.